snuba/web/delete_query.py

```diff
@@ -27,23 +32,62 @@ def construct_condition(body: Dict[str, Any]) -> Expression:
     return combine_and_conditions(and_conditions)
 
 
+def _get_rows_to_delete(
+    storage_key: StorageKey, select_query_to_count_rows: Query
+) -> int:
+    formatted_select_query_to_count_rows = format_query(select_query_to_count_rows)
+    select_query_results = (
+        get_storage(storage_key)
+        .get_cluster()
+        .get_reader()
+        .execute(formatted_select_query_to_count_rows)
+    )
+    return typing.cast(int, select_query_results["data"][0]["count"])
+
+
+def _enforce_max_rows(delete_query: Query) -> None:
```

Comment:
leave a short comment explaining what this is for and why. It doesn't have a return type so the user has to glean from the thrown errors why this was every implemented